seeds/chainlink/_schema.yml

```diff
@@ -0,0 +1,69 @@
+version: 2
+
+seeds:
+  - name: chainlink_bnb_price_feeds
```

Comment:
this must be the name of the seed file, not model name -- is that set correctly?